SmartTutor/ContentModel/SmartTutorContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace SmartTutor.ContentModel
+{
+    public class SmartTutorContext : DbContext
+    {
+        private const string CONNECTION_STRING = "User ID =postgres;Password=super;Server=localhost;Port=5432;Database=smart-tutor-db;Integrated Security=true;Pooling=true;";
```

Comment:
It would be better if we could load the connection string from configuration. This article can help https://docs.microsoft.com/en-us/ef/core/miscellaneous/connection-strings

Edit: I see that you are configuring the connection string in the Startup.cs, but I'm not sure what's going on here then? We'll discuss this.